resources/lang/es/passwords.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/*
+|--------------------------------------------------------------------------
+| Password Reset Language Lines
+|--------------------------------------------------------------------------
+|
+| The following language lines are the default lines which match reasons
+| that are given by the password broker for a password update attempt
+| has failed, such as for an invalid token or invalid new password.
+|
+*/
+
+return [
+    'reset'     => '¡Su contraseña ha sido restablecida!',
+    'sent'      => '¡Le hemos enviado por correo electrónico el enlace para restablecer su contraseña!',
+    'throttled' => 'Por favor espere antes de intentar de nuevo.',
```

Comment:
when is this message shown?

Reply:
Es para la sobrecarga de envios atravez de las rutas.